dapps/dapps_dev.json

```diff
@@ -159,22 +159,6 @@
         "staking"
       ]
     },
-    {
-      "name": "Bit.Country Pioneer Rewards Claim",
-      "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Bit.Country.svg",
-      "url": "https://ksmcrowdloan.bit.country/reward",
-      "categories": [
-        "crowdloans"
-      ]
-    },
-    {
-      "name": "Parallel Rewards Claim",
-      "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/color/Parallel-Heiko.svg",
-      "url": "https://crowdloan.parallel.fi/",
-      "categories": [
-        "crowdloans"
-      ]
```

Comment:
Looks like this one still can be kept, they provide their own interface for crowdloans, but need to change the name.
<details>
  <summary>proof</summary>

<img width="1388" alt="Screenshot 2022-12-01 at 10 50 16" src="https://user-images.githubusercontent.com/40560660/204976205-59fcfbd3-4a71-4b05-8ccf-c5f29c2d636f.png">

</details>

Reply:
fixed